apps/site/hooks/react-client/useClientContext.ts

```diff
@@ -2,8 +2,8 @@
 
 import { useContext } from 'react';
 
-import { MatterContext } from '@/providers/matterProvider';
-import type { ClientSharedServerContext } from '@/types';
+import { MatterContext } from '../../providers/matterProvider';
```

Comment:
I'm not sure I ever saw an answer to why a bunch of these have become complex relative imports instead of using the `@/` alias?

Reply:
Ack! I must've missed the .ts(x) files!

The reason for the change (will I will revert) is that "@/" means different things depending on what's importing it. There's more details on Slack, but it doesn't really affect this PR, so I'll revert.

I'll link the discussion here later today.